README.md

```diff
@@ -22,8 +22,6 @@ It syncs your LaunchDarkly environment to your Convex deployment, allowing you t
 
 To use the LaunchDarkly Convex component, you'll need a LaunchDarkly account. Your LaunchDarkly subscription must include access to integrations.
 
-Copy your LaunchDarkly environment's SDK Key and store it as an environment variable named `LAUNCHDARKLY_SDK_KEY` in your Convex deployment. You can do so on the [environment variables](https://dashboard.convex.dev/deployment/settings/environment-variables) page or via `npx convex env set LAUNCHDARKLY_SDK_KEY <key>` from the CLI.
```

Comment:
This needs to move down because you need to have the component set up before adding the SDK key now